apps/src/dance/lab2/views/generate-dance.module.scss

```diff
@@ -22,3 +22,12 @@
     font-size: 16px;
   }
 }
+
+.hiddenButton {
```

Comment:
nit: can make this a mixin that you just `@include` in all these files to save on repetition. Or even a react component

Reply:
Good idea; might save this for a follow-up.  Any idea of a good place to put a common SCSS file?

Reply:
We could do within Lab2 or `apps/src/mixins`? If this is a pattern we don't mind repeating, creating a common `<HiddenFocusButton>` could be useful too, though I'd defer to @hannahbergam